enhancements/olm-admission-webhooks.md

```diff
@@ -106,6 +106,8 @@ It important to note that the Webhook structure does not provide users with the
 - OLM will create the WebhookConfigClient after generating the Service and the CA-Cert.
 - The NamespaceSelector field will be generated based on the OperatorGroup that the operator is deployed in. When creating the OperatorGroup, OLM will apply a label to all namespaces that are included in that operator group. Then, when creating the Webhook, OLM will update the namespace selector to use that label.
 
+It is also important to call out that OLM will not allow webhooks to define rules that allows the webhook to intercept actions against CSVs and webhooks. Consider the worst case scenario where a broken webhook intercepts all calls against any resource and has a `Fail` [FailurePolicy](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#failure-policy). In this scenario, all calls to the K8s API would be rejected. As such OLM will implement constraints that prevent webhooks from including rules that include webhooks. Furthermore, if a webhook includes a rule that allows it to intercept calls to delete a CSV, it could effectively prevent a cluster admin from disabling the webhook described earlier, as OLM would attempt to reinstall the deleted webhook. As such, OLM will not allow webhooks to include rules that allows the webhook to intercept API requests targeting CSVs.
```

Comment:
Can/should we prevent these to be added to the registry or during bundle validation?

Reply:
I believe this suggestion would make the process of creating a first bundle easier for Operator Authors and improve the overall user experience.